types/wordpress__blocks/api/registration.d.ts

```diff
@@ -20,7 +20,7 @@ export function getBlockSupport<T>(
  * Returns a registered block type.
  */
 // tslint:disable:no-unnecessary-generics
-export function getBlockType<T extends Record<string, any> = any>(name: string | undefined): Block<T> | undefined;
+export function getBlockType<T extends Record<string, any> = {}>(name: string | undefined): Block<T> | undefined;
```

Comment:
Was this change necessary?

Reply:
Oh, it wasn't, it's just a leftover after merging #59550